### ISSUE TITLE
Modify Training Epochs from 12 to 50 for Strip RCNN-S Model

### ISSUE THREAD

**Issue by Bhavana431** [HVision-NKU/Strip-R-CNN#15]
I have trained the Strip RCNN-S model for 12 epochs, but the accuracy achieved is only 56%. I would like to increase the number of training epochs from 12 to 50 to potentially improve the model’s performance.

Could you please guide me on where to make this change in the code?

Thank you for your support.

**Comment by getup201** [comment]
`runner = dict(type='EpochBasedRunner', max_epochs=50)`

Add this piece of code to the config file

**Comment by YXB-NKU** [comment]
Hello, you can refer to the issue https://github.com/HVision-NKU/Strip-R-CNN/issues/5 for help. There might be some other issues with your training. Relying solely on increasing the number of epochs is likely not going to solve the problem .

**Comment by Bhavana431** [comment]
@YXB-NKU 

https://captain-whu.github.io/DOTA/dataset.html

I initially obtained the dataset with the above link is :
Training set: 1,411 images
Validation set: 458 images
Testing set: 937 images

I performed a single split and updated the dataset to:
Training set: 15,749 images
Validation set: 5,297 images
Testing set: 10,833 images

Is this new split ratio suitable and sufficient for effectively training the STRIP-RCNN model?

**Comment by Bhavana431** [comment]
Hello @YXB-NKU,

I referred to the link https://github.com/HVision-NKU/Strip-R-CNN/issues/5 and made modifications accordingly. However, I only achieved a 69% mAp. I have attached the log file. Could you please check it and help me identify where I might have gone wrong?

https://rangsonstech-my.sharepoint.com/:u:/g/personal/bhavanabc_rangsonsaerospace_com/EedfKI_uKkJEvj-Z4Ng5XeoBX7YHpVKILPxyavsGs4pZ0g?e=1UJK20

**Comment by YXB-NKU** [comment]
It seems that you have attached a .pth file instead of a log file. Please double-check if the dataset size is correct and whether the learning rate and batch size (bs) correspond to each other .

**Comment by Bhavana431** [comment]
@YXB-NKU 
Sorry for Inconveniences

https://rangsonstech-my.sharepoint.com/:u:/g/personal/bhavanabc_rangsonsaerospace_com/EWj7x_ZQzBhJjmRLzyibmy4B0M7TS1Qtd1lBhdsZQNFDxA?e=T84YZo

**Comment by YXB-NKU** [comment]
It seems that the model was trained using the single-scale DOTA dataset. You can refer to this log; there doesn't appear to be any issues with it .

**Comment by YXB-NKU** [comment]
https://github.com/HVision-NKU/Strip-R-CNN/issues/18

**Comment by Bhavana431** [comment]
Hello , @YXB-NKU 

I want to train the model with a different dataset. In that case, is there any way to convert annotation text files into the DOTA format, or do I have to create them manually? Also, which website can I refer to for annotation tools or guidelines?

**Comment by Bhavana431** [comment]
Hello, @YXB-NKU

I am currently training the model by doing the following:
I take the dataset and split it using single scale into:
train + val = trainval → used for training
val → used for validation
test → used for testing

Is this method correct, or should I do the following instead:
Take the full trainval dataset, then split it into:
80% for training
20% for validation

Which one is the correct and recommended approach?

**Comment by YXB-NKU** [comment]
use this one:
train + val = trainval → used for training
val → used for validation
test → used for testing

**Comment by Bhavana431** [comment]
@YXB-NKU 
got only 76% accuracy why?
https://rangsonstech-my.sharepoint.com/:u:/g/personal/bhavanabc_rangsonsaerospace_com/EbNoGiGxeBBGvUiLdJSZ9aQBHOn07r66NsNAU_RELNZ4JQ?e=LIggRo
